### PR TITLE
Reject standalone checks in API before serving "/checks" routes

### PR DESCRIPTION
**Pull request by amdprophet**
## Description
The `/checks` & `/check/:check` routes in Sensu API should only show subscription checks, not standalone checks.

## Related Issue
https://github.com/sensu/sensu/issues/1416

## Motivation and Context
Standalone checks will be picked up by Sensu API if a Sensu Client exists on the same node and they both load checks from the same configuration file and/or directories. Including standalone checks that only exist on the node running Sensu API may be misleading to users as they may expect the API to also show information on standalone checks that exist outside of the API server.

## How Has This Been Tested?
Wrote test specs. Ensured they failed prior to the fix and that they passed after the fix.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [x] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

File: lib/sensu/api/routes/checks.rb [hunk 1]
```diff
@@ -7,14 +7,14 @@ module Checks
 
         # GET /checks
         def get_checks
-          @response_content = @settings.checks
+          @response_content = @settings.checks.reject { |c| c[:standalone] }
```
Comment (portertech) [inline lib/sensu/api/routes/checks.rb]: `c` -> `check`

**Comment by portertech** [comment]
@amdprophet looks good, please target release/0.26

**Comment by fessyfoo** [comment]
tx for this. 😄

**Comment by cwjohnston** [comment]
Closes #1416

**Pull request merged**